### ISSUE TITLE
Add windows support please

### ISSUE THREAD

**Issue by tzwilliams** [pytorch/vision#252]
I am wondering if anyone is developing a win64 branch of torchvision.  
I've been following the conversation regarding [installing pytorch on Windows](https://github.com/pytorch/pytorch/issues/494), and I have @peterjc123 version of PyTorch running successfully (thanks, @peterjc123!).

**Comment by peterjc123** [comment]
@tzwilliams You can install torchvision through pip. Just `pip install torchvision` is fine.

**Comment by alykhantejani** [comment]
@tzwilliams torchvision should work just fine on windows (provided you can get pytorch running)

**Comment by fmassa** [comment]
Given that torchvision is a python-only library, it should work just fine on Windows.
I'm closing the issue, but let me know if you face any problems running it on Windows.

**Comment by tzwilliams** [comment]
Thanks everyone.  I had previously tried `conda install torchvision` and been told that there was no win64 build available.  Of course, I should have also tried `pip`!  It looks like everything is up and running now. Thanks for the help.

**Comment by Sudhakar17** [comment]
![capture1](https://user-images.githubusercontent.com/18436807/32890916-0763c20c-cad1-11e7-9ebc-99c10225e024.PNG)

Hi tzWilliams,

I am getting this error while installing torchvision in windows. Can you please share how did you install it ? I am using pytorch in vitualenv using conda.

**Comment by alykhantejani** [comment]
@Sudhakar17 is pytorch already installed before running this?

**Comment by tzwilliams** [comment]
@Sudhakar17 I've never used virtualenv, but I'm happy to share the notes I took back when I did get pytorch and torchvision working.  

Directly answering your question about torchvision, once I had pytorch setup in Windows the torchvision install was successfully completed with `pip install torchvision`

It sounds like you don't need this piece, but for installing pytorch in Windows I first followed the instructions at [https://www.superdatascience.com/pytorch/](url).  I later installed the updated version of pytorch available at [https://anaconda.org/peterjc123/pytorch/files](url)

I hope that helps!  Good luck.

**Comment by stepanveret** [comment]
This helped for me
1. `conda install -c peterjc123 pytorch`
2. `pip install torchvision`

**Comment by rajarsheem** [comment]
@stepanveret Does it have any cons?

**Comment by Bestiplays** [comment]
@Sudhakar17 
Hi, I have the same problem then you. Did you could solve it?! It would be very nice if anyone could help me. I have installed pytorch successfully using anaconda, but can't install torchvision.

Thank you.

**Comment by fmassa** [comment]
@Bestiplays I don't have experience with Windows, but torchvision is a python-only package, if you can't install it via pip, you could clone the repo and do something like
```
python setup.py install
```
from the cloned torchvision folder (with the proper translation to Windows commands, that I unfortunately don't know)

**Comment by AKABBOKSIL** [comment]
@Bestiplays stepanveret's method does work!
you should do it on Anaconda prompt

**Comment by peterjc123** [comment]
@GANESHKUMAR7860 Can you post the full traceback here? I guess the problem is on PIL, not torchvision.

**Comment by GANESHKUMAR7860** [comment]
@peterjc123 please give some solutions for this error. Thank you 
Traceback (most recent call last):
  File "<pyshell#3>", line 1, in <module>
    import torchvision
  File "C:\Users\A.GANESH KUMAR\Anaconda3\lib\site-packages\torchvision\__init__.py", line 2, in <module>
    from torchvision import datasets
  File "C:\Users\A.GANESH KUMAR\Anaconda3\lib\site-packages\torchvision\datasets\__init__.py", line 1, in <module>
    from .lsun import LSUN, LSUNClass
  File "C:\Users\A.GANESH KUMAR\Anaconda3\lib\site-packages\torchvision\datasets\lsun.py", line 2, in <module>
    from PIL import Image
  File "C:\Users\A.GANESH KUMAR\Anaconda3\lib\site-packages\PIL\Image.py", line 58, in <module>
    from . import _imaging as core
ImportError: DLL load failed: The specified module could not be found.

**Comment by GANESHKUMAR7860** [comment]
@peterjc123 
When i import the torchvision in jupyter notebook . It shows some error and i have installed the pillow also.  solutions please?
ImportError                               Traceback (most recent call last)
<ipython-input-1-6dd351122000> in <module>()
----> 1 import torchvision

~\Anaconda3\lib\site-packages\torchvision\__init__.py in <module>()
      1 from torchvision import models
----> 2 from torchvision import datasets
      3 from torchvision import transforms
      4 from torchvision import utils
      5 

~\Anaconda3\lib\site-packages\torchvision\datasets\__init__.py in <module>()
----> 1 from .lsun import LSUN, LSUNClass
      2 from .folder import ImageFolder, DatasetFolder
      3 from .coco import CocoCaptions, CocoDetection
      4 from .cifar import CIFAR10, CIFAR100
      5 from .stl10 import STL10

~\Anaconda3\lib\site-packages\torchvision\datasets\lsun.py in <module>()
      1 import torch.utils.data as data
----> 2 from PIL import Image
      3 import os
      4 import os.path
      5 import six

~\Anaconda3\lib\site-packages\PIL\Image.py in <module>()
     58     # Also note that Image.core is not a publicly documented interface,
     59     # and should be considered private and subject to change.
---> 60     from . import _imaging as core
     61     if PILLOW_VERSION != getattr(core, 'PILLOW_VERSION', None):
     62         raise ImportError("The _imaging extension was built for another "

ImportError: DLL load failed: The specified module could not be found.

**Comment by GANESHKUMAR7860** [comment]
Excuse me any solution for the above error please.

**Comment by peterjc123** [comment]
@GANESHKUMAR7860 Would you please uninstall `torchvision` and `Pillow` and then reinstall them?

**Comment by GANESHKUMAR7860** [comment]
@peterjc123 
I did it but nothing works. Thank's.

**Comment by sunnydps32** [comment]
**conda install pytorch-cpu -c pytorch 
pip3 install torchvision**

I was facing a similar problem earlier. The first command was running fine. But the second one was showing error.
There were two installations of pip. One was inside the installation directory of python and the other was inside Miniconda. This was causing the conflict.
I removed the pip from the installation directory of python using the following command :-
**python -m pip uninstall pip**

After that, the second command ran successfully.
Then, I was able to do "import torch" and "import torchvision" from Anaconda Prompt.
